UI/window-basic-main.cpp

```diff
@@ -152,24 +151,25 @@ static void AddExtraModulePaths()
 
 	char base_module_dir[512];
 #if defined(_WIN32) || defined(__APPLE__)
-	int ret = GetProgramDataPath(base_module_dir, sizeof(base_module_dir),
-				     "obs-studio/plugins/%module%");
+	int ret =
+		GetProgramDataPath(base_module_dir, sizeof(base_module_dir),
+				   "light-studio/obs-studio/plugins/%module%");
```

Comment:
This seems unrelated to the PR. There are 13+ instances of this change.

Reply:
Thanks for catching this, I didn't mean for this change to be included. I should have marked PR as a draft.